src/components/SignUp/index.scss

```diff
@@ -0,0 +1,98 @@
+@import "../../style/variables";
+@import "../../style/buttons";
+@import "../../style/headings";
+@import "../../style/bootstrapImports";
+
+.signUpSection {
+  /* All section style */
+  width: 100%;
+  margin-top: 4rem;
+  margin-bottom: 4rem;
+  background-color: $lightColor;
+
+  .signInImg {
```

Comment:
signup vs signin here again. if you are copy-pasting code from sign-in then i'd consider finding a way to reuse the same CSS here.

Reply:
I will try to find a way as you suggest